modules/mobile/app/serializers/mobile/v0/user_serializer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'fast_jsonapi'
+
+module Mobile
+  module V0
+    class UserSerializer
+      include FastJsonapi::ObjectSerializer
+
+      # TODO: return full set of user data, including latest appeals, claims, and appointments
```

Comment:
Is this `TODO` before the PR is merged, or is there a follow-up ticket for traceability purposes?

Reply:
this is for follow up, wanted a slimed down version to test auth before the full user endpoint goes in another PR

Reply:
added and issue # to the todo